watertap/examples/flowsheets/ion_exchange/ion_exchange_demo.py

```diff
@@ -393,10 +303,10 @@ def display_results(m):
         f'{"TOTAL Capital Cost":<40s}{f"${ix.costing.capital_cost():<39,.2f}"}{"$":<40s}'
```

Comment:
Do you use the `report` method at all for IX? It could be added here potentially.

Reply:
I typically don't because I want different units than the Vars are in, so I have made my own reporting functions.